dxr/lines.py

```diff
@@ -321,12 +317,12 @@ def nesting_order((point, is_start, payload)):
                              -payload.sort_order)
 
 
-def finished_tags(text, refs, regions):
+def finished_tags(lines, refs, regions):
     """Return an ordered iterable of properly nested tags which fully describe
     the refs and regions and their places in a file's text.
 
-    :arg text: Unicode text of the file to htmlify. ``build_lines`` may not be
-        used on binary files.
+    :arg lines: iterable of lines of text of the file to htmlify.
+        ``build_lines`` may not be used on binary files.
```

Comment:
Heh, might as well delete this line (as it's clear enough already). build_lines() doesn't exist anymore.